debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/RecordsSnapshotProducerIT.java

```diff
@@ -70,7 +70,7 @@ public void after() throws Exception {
     public void shouldGenerateSnapshotsForDefaultDatatypes() throws Exception {
         snapshotProducer = new RecordsSnapshotProducer(context, new SourceInfo(TestHelper.TEST_SERVER, TestHelper.TEST_DATABASE), false);
 
-        TestConsumer consumer = testConsumer(ALL_STMTS.size(), "public", "Quoted_\"");
+        TestConsumer consumer = testConsumer(ALL_STMTS.size(), "public", "Quoted_");
```

Comment:
Probably nitpicking but should not be there double underscore - `Quoted__`? Both would work but the socnd one is similar to the value replaced - the specific one.

Reply:
Good catch, this is a left-over from some intermediary work state. I'll adjust as you suggest.